### PR TITLE
Add marked maker

### PR DESCRIPTION
**Pull request by joshbruce**
**Marked version:** 0.3.17

**Markdown flavor:** n/a

## Description

This PR adds a new author badge and gives first recognition to @carlosvalle.

## Contributor

- [ ] Test(s) exist to ensure functionality and minimize regresstion (if no tests added, list tests covering this PR); or,
- [x] no tests required for this PR.
- [ ] If submitting new feature, it has been documented in the appropriate places.

## Committer

In most cases, this should be a different person than the contributor.

- [ ] Draft GitHub release notes have been updated.
- [ ] CI is green (no forced merge required).
- [ ] Merge PR

### CODE REVIEW

File: assets/marked.svg [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (styfle) [inline assets/marked.svg]: Instead of `/assets/marked.svg` can you put this in `/docs/img/marked.svg`

Comment (styfle) [reply assets/marked.svg]: Then add `docs` to the `.npmignore` file.

**Comment by joshbruce** [comment]
@styfle: Thanks. Good catch. Was actually thinking of closing anyway - should have been two PRs. One to add the badge, the other to nominate someone.

**Comment by styfle** [comment]
@joshbruce Agreed

**Pull request closed without merging**